ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -534,3 +536,117 @@ def test_userroot_ldif_files_ownership(self):
         unexp_str = "CRITICAL: db2ldif failed:"
         assert cmd.returncode == 0
         assert unexp_str not in cmd.stdout_text
+
+
+class TestBackupAndRestoreDMPassword(IntegrationTest):
```

Comment:
Hi @mrizwan93 
The test TestBackupAndRestoreDMPassword is linked to a different issue and should not be part of this backport (see https://pagure.io/freeipa/issue/7136). Can you remove it from your commit?